### ISSUE TITLE
Accessibility - Submit Screen - VoiceOver Should Announce Asterisks as "Required"

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-iOS#45]
The asterisks for the required fields should be announced as “Required” instead of a star.

**Comment by aanthonyDOL** [comment]
Fixed

**Comment by jfc3-dol** [comment]
Thanks, will test in the next version of the application.

**Comment by jfc3-dol** [comment]
VoiceOver is reading all the asterisks as "Required" instead of "star", so I'm going to close this issue because it is now working as expected.
